src/panel/poi_bloc/poi_bloc_container.js

```diff
@@ -1,14 +1,28 @@
+/* global require */
 import Panel from '../../libs/panel';
 import PoiBlocContainerView from '../../views/poi_bloc/poi_bloc_container.dot';
 import constants from 'config/constants.yml';
+import ReactDOMServer from 'react-dom/server';
+import React from 'react';
 
 function PoiBlocContainer() {}
 
 PoiBlocContainer.initBlockComponents = function() {
   PoiBlocContainer.blockComponents = constants.pois.reduce((accBlocks, poiBlock) => {
+    let builder;
+    try {
+      builder = require(`./${poiBlock.panelName}_panel`);
+    } catch (err) {
+      const name = poiBlock.panelName.charAt(0).toUpperCase() + poiBlock.panelName.slice(1);
+      const el = require(`../../views/poi_bloc/${name}`);
+      builder = {
+        default: (x, y, z) => ReactDOMServer.renderToString(<el.default x={x} y={y} z={z} /> ),
```

Comment:
For this case it's better to use `renderToStaticMarkup` as it produces cleaner DOM https://reactjs.org/docs/react-dom-server.html#rendertostaticmarkup
And even better, using `src/libs/renderStaticReact` which does that for you by hiding the dirty detail, as using the `ReactDOMServer` on the client is really a hack.